docs/rules/new-ember-data-packages.md

```diff
@@ -0,0 +1,41 @@
+# Use &#34;Ember Data Packages&#34; from Ember RFC #395 (new-ember-data-packages)
+
+The goal of this rule is to ease the migration to the new @ember-data packages.
+
+## Context
+
+`ember-data has been split in multiple packages. For instance, its store is now released in "@ember-data/store" package. These packages have been released starting from ember-data version 3.11.
+
+## Rule Details
+
+Examples of **incorrect** code for this rule:
+
+```js
+import Model from "ember-data/model";
+import attr from "ember-data/attr";
+
+import DS from "ember-data";
+const { Model } = "ember-data";
+```
+
+Examples of **correct** code for this rule:
+
+```js
+import Model, { attr } from "@ember-data/model";
+
+import Model from "@ember-data/model";
+```
+
+## How to fix
+
+This rule implements a fix function (or to be more precise: it leverages the `no-old-shims` fix function). If, as a user, you can get rid of all the `import DS from "ember-data";` imports, you'll be able to use this rule fixer to complete the upgrade. 
+
+Note that a codemod is also available to complete the upgrade.
+
+## When Not To Use It
+
+You don't want to use this rule if you're using `ember-data@<3.11`.
```

Comment:
just making a note to link any polyfill we create for older versions here in the future, nothing actionable at this time

Reply:
👍

Reply:
link to issue: https://github.com/emberjs/data/issues/6266